03/Product.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+class Product extends React.Component {
+	handleClick = (e, product) => {
+		const { isCategory, isCart } = this.props
+		e.preventDefault()
+		isCategory ? this.props.add(product) : isCart ? this.props.remove(product) : ''
```

Comment:
Zastanawiam się czy faktycznie podwójnie musimy to sprawdzać.
Jeśli nie jest to kategoria to przypadkiem nie będzie to koszyk (bez sprawdzania)?